### ISSUE TITLE
OOM error

### ISSUE THREAD

**Issue by laybebe** [SysCV/MaskFreeVIS#4]
Thanks for your excellent work! What's the GPU memory of your Titan RTX, 24G?  I have tried to train MaskFreeVIS with backbone ResNet50  and batch size 16 on 8x3090(24G), but it will result in an OOM error.

**Comment by lkeab** [comment]
which training script are you using?

**Comment by laybebe** [comment]
> which training script are you using?

I used "MaskFreeVIS/mfvis_nococo/configs/youtubevis_2019/video_maskformer2_R50_bs16_8ep.yaml" and did not modify any parameter.

**Comment by lkeab** [comment]
In this case, you can reduce the [SAMPLING_FRAME_NUM](https://github.com/SysCV/MaskFreeVIS/blob/main/mfvis_nococo/mask2former_video/config.py#L9) from 5 to 3, and modify the codes [here](https://github.com/SysCV/MaskFreeVIS/blob/main/mfvis_nococo/mask2former_video/modeling/criterion.py#L338) accordingly (to make TK loss work in 3-frame tube). It will reduce memory a lot but the same time training will be slightly less stable.

**Comment by laybebe** [comment]
Thanks for your reply. In fact, I ran the code by reducing the batch size. The result (AP=41.7) was slightly lower than the result in the paper.

**Comment by lkeab** [comment]
Yeah, that's actually normal due to the weak supervision (no any mask usage) and the randomness in sampling frames of the dataloader.
